python/cudf/cudf/core/column/column.py

```diff
@@ -1185,7 +1183,7 @@ def as_column(arbitrary, nan_as_null=None, dtype=None, length=None):
         if dtype is not None:
             data = data.astype(dtype)
     # TODO: Remove nvstrings here when nvstrings is fully removed
-    elif isinstance(arbitrary, nvstrings.nvstrings):
+    elif type(arbitrary).__name__ == "nvstrings":
```

Comment:
Can we move this condition lower down below the `__cuda_array_interface__` condition? This way for other GPU objects it can short circuit this check.